ios/ReactNativeConfig/ReadDotEnv.rb

```diff
@@ -20,6 +21,12 @@ def read_dot_env(envs_root)
     file = ENV['ENVFILE'] || defaultEnvFile
   end
 
+  if File.exists?("/tmp/useenvlocal")
```

Comment:
need help: where is the proper place to remove this file, since it won't be overwritten like `/tmp/envfile`?
This PR has been opened before this changes https://github.com/luggit/react-native-config/commit/ab8021cb2b31a91353eae02288cbf7325a78fa6a